brainzutils/musicbrainz_db/event.py

```diff
@@ -93,3 +94,57 @@ def fetch_multiple_events(mbids, includes=None):
             )
 
         return {str(mbid): serialize_events(event, includes_data[event.id]) for mbid, event in events.items()}
+
+
+def get_event_for_place(place_id, event_types=None, limit=None, offset=None):
+    """Get all events linked to a place.
+
+    Args:
+        place_id (uuid): MBID of the place.
+        event_types (list): List of types of events to be fetched.
+        limit (int): Max number of events to return.
+        offset (int): Offset that can be used in conjunction with the limit.
+
+    Returns:
+        Tuple containing the list of dictionaries of events ordered by begin date
+        and the total count of the events.
+    """
+
+    place_id = str(place_id)
+    if event_types is None:
+        event_types = []
+    event_types = [event_type.lower() for event_type in event_types]
+
+    includeNULL = False
+    if 'none' in event_types:
+        includeNULL = True
+        event_types.remove('none')
+    
+    # map event types to their case sensitive name in musicbrainz.event_type table in the database
+    event_types_mapping = {
+        'award ceremony': 'Award ceremony',
+        'concert': 'Concert',
+        'convention/expo': 'Convention/Expo',
+        'festival': 'Festival',
+        'launch event': 'Launch event',
+        'masterclass/clinic': 'Masterclass/Clinic',
+        'stage performance': 'Stage performance',
+    }
```

Comment:
As in https://github.com/metabrainz/brainzutils-python/pull/86/files#r862862233, get this list of types fromthe database

Reply:
Done